### PR TITLE
Add syndicated version of Support Solidus Article from Karma Creative

### PR DESCRIPTION
**Pull request by MFRWDesign**
This PR adds a new post for the syndicated version of the Support Solidus Article.

@aldesantis @kennyadsl I know we've been talking about some upcoming changes; please let me know if you feel this and/or the original article needs any update (now or in the near future to reflect our current policies).

### CODE REVIEW

**Comment by aldesantis** [comment]
I think it's fine as is — if needed, we can revise it in the future or, even better, post an update.

File: source/blog/2020-06-24-support-solidus.html.markdown [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+Since Stembolt left, we’ve been busy. And creating a Slack channel was just the first step of many. From there, we set up organizational meetings, assigned roles, created streams of funding through recurring donations to the platform, supported and learned from each other, organized and attended conferences and, most importantly, built really solid relationships and trust within the community.
+
+And as we touched on in our talk, trust and good relationships allow us to move forward together. Without it, there’s no real way to have an honest conversation about the roadmap, next steps, collaboration, or really anything.
+
+In summary, the Support Solidus initiative was meant, first and foremost, as a *call to arms*. With your code, through becoming an Ambassador, or by donating your time, we want to remind everyone that, together, it can be done.
+
+---
+_To continue reading this article, please visit the original on the [Karma Creative](https://karmacreative.io) website: [Full Text of original Support Solidus Article](https://karmacreative.io/support-solidus)._
```
Comment (aldesantis) [inline source/blog/2020-06-24-support-solidus.html.markdown:101]: I'm a bit concerned by the UX implications of asking people to visit another site to continue reading, although I'm on expert here and don't really have a strong opinion. @kennyadsl what are your thoughts?

Comment (kennyadsl) [reply source/blog/2020-06-24-support-solidus.html.markdown:101]: I don't have any specific concerns around continue reading on an external website but for me, the reasoning behind this is just not clear. If it's for SEO reasons, we have the canonical tag and should not impact.

**Comment by aldesantis** [comment]
@MFRWDesign can you update the PR (including the date of the post) and then we can merge?

**Comment by MFRWDesign** [comment]
@aldesantis Sorry for the massive delay here - just got all tied up with client work and these changes kept getting pushed down in the stack. This should again be ready for review.

I changed the CTA at the bottom of the syndicated version of the article to point to Open Collective, and I updated the publication date.

(FYI My rationale for not including the whole article was the it was extremely long and didn't fit the flow of the rest of the Solidus blog, not to mention it would've required adding a bunch more pictures, adapting styles, and creating a ToC element. -- We'll circle back around on this in the future and can come back to make an update here when we have time.)

**Review by kennyadsl — APPROVED** [review]
Works for me, thanks @MFRWDesign !

**Pull request merged**